web/docs/auth/auth-hooks.md

```diff
@@ -520,6 +528,54 @@ export const onAfterLogin: OnAfterLoginHook = async ({
 
 Read more about the data the `onAfterLogin` hook receives in the [API Reference](#the-onafterlogin-hook).
 
+### Refreshing the OAuth access token
+
+If you are using OAuth, you can use the `onAfterSignup` or `onAfterLogin` hooks to refresh the OAuth access token if the provider supports it.
+
+You'll need to import the OAuth client from the `wasp/server/oauth` module for the provider you are using. The OAuth client has a `refreshAccessToken` method that you can use to refresh the access token.
```

Comment:
```suggestion
Import the OAuth client for the desired provider from `wasp/server/oauth` and use its `refreshAccessToken` method to refresh the access token.
```
Suggestions based on:
 - Addressing the user directly
 - Prefer present tense
 - Prefer shorter and simpler sentences